weis/aeroelasticse/openmdao_openfast.py

```diff
@@ -112,9 +108,42 @@ def setup(self):
         n_freq_blade = int(rotorse_options['n_freq']/2)
         n_pc         = int(rotorse_options['n_pc'])
 
-        FASTpref = self.options['modeling_options']['openfast']
-        # self.FatigueFile   = self.options['modeling_options']['rotorse']['FatigueFile']
-        
+        # DLC options
+        n_OF_dlc11 = self.options['modeling_options']['DLC_driver']['n_cases_dlc11']
+
+        # OpenFAST options
+        OFmgmt = self.options['modeling_options']['DLC_driver']['openfast_file_management']
+        self.model_only = OFmgmt['model_only']
+        FAST_directory_base = OFmgmt['OF_run_dir']
+        # If the path is relative, make it an absolute path
+        if not os.path.isabs(FAST_directory_base):
+            FAST_directory_base = os.path.join(os.getcwd(), FAST_directory_base)
+        # Flag to clear OpenFAST run folder. Use it only if disk space is an issue
+        self.clean_FAST_directory = False
+        self.FAST_InputFile = OFmgmt['OF_run_fst']
+        # File naming changes whether in MPI or not
+        if MPI:
+            rank    = MPI.COMM_WORLD.Get_rank()
+            self.FAST_directory = os.path.join(FAST_directory_base,'rank_%000d'%int(rank))
+            self.FAST_namingOut = self.FAST_InputFile+'_%000d'%int(rank)
+        else:
+            self.FAST_directory = FAST_directory_base
```

Comment:
I think we used to have `FAST_runDirectory` here; this is causing errors further down.  Do we want `FAST_directory` for any particular reason instead? Before, that was the base directory of the original model.

Reply:
@ptrbortolotti will have to answer that one when he gets a chance

Reply:
I ended up changing this back to `FAST_runDirectory` because that's how `runFAST_pywrapper` is set up and it didn't look like there are changes there along these lines.

Reply:
I'm going to check this later today because I'm pretty sure things were working fine. Updates will follow

Reply:
@dzalkind right now we have FAST_directory and FAST_runDirectory and maybe we don't need both? I can clean this up on Monday myself

Reply:
I don't think `openmdao_openfast.py` uses `FAST_directory` that much, but I have some updates down the road that will use it some. `FAST_directory` an attribute of `runFAST_pywrapper` that defines the base directory of a given OpenFAST model; I'd like to keep that in there.